Sources/TensorFlow/Layers/Core.swift

```diff
@@ -223,3 +223,22 @@ public extension Dense {
                   activation: activation)
     }
 }
+
+
+public struct Function<Input: Differentiable, Output: Differentiable>: Layer {
+    /// The Function Layer arbitrarily takes any differentiable function and returns a Layer which can
```

Comment:
This comment does not accurately describe the layer in that
- A layer does not _take_ a differentiable function, it _stores_ a differentiable function.
- A layer does not _return_ any layer. What returns is its `callAsFunction(_:)` method.

Also, the function layer is not a proper name, and thus should not be spelled as "Function Layer". In API documentation, it should be spelled as "The `Function` layer" with the word "layer" starting with a lower case `l`.

Moreover, this is an API summary. It should be placed on top of the declaration of the `Function` type, not within.

Reply:
@rxwei Yes, I was going to ask some comments on the documentation. I am just getting the code right.